charts/aws-efs-csi-driver/values.yaml

```diff
@@ -41,6 +41,8 @@ controller:
   create: true
   # Number for the log level verbosity
   logLevel: 2
+  # If set, add pv/pvc metadata to plugin create requests as parameters.
+  extraCreateMetadata: false
```

Comment:
we can default it true, i ended up doing so in ebs https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/937

Reply:
I was not sure about that. I'll do it.

Reply:
Done